drainer/loopbacksync/loopbacksync.go

```diff
@@ -16,6 +16,8 @@ package loopbacksync
 const (
 	//MarkTableName mark table name
 	MarkTableName = "retl._drainer_repl_mark"
+	//ID syncer worker thread id
+	ID = "id"
```

Comment:
how about change to a meaningful name like `threadID`?

Reply:
This is not a real thread ID, just a 0-workercount-1 sequence for go's concurrent programs；Maybe it's called coroutineID is appropriate？